packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/pages/home/index.jsx.hbs

```diff
@@ -57,6 +57,16 @@ const Home = () => {
     const {data: productSearchResult, isLoading} = useProductSearch({
         parameters: {
             refine: [`cgid=${HOME_SHOP_PRODUCTS_CATEGORY_ID}`, 'htype=master'],
+            perPricebook: true,
+            allVariationProperties: true,
+            expand: [
+                'availability',
```

Comment:
Do we need `promotions` expansion?

Reply:
Good catch, we probably should include it here